Dockerfile

```diff
@@ -60,7 +69,6 @@ RUN set -eux; \
     sudo chmod +x /usr/local/bin/datadog-ci;\
     sudo rm -rf /tmp/..?* /tmp/.[!.]* /tmp/*;
```

Comment:
Is it still required as we do no more install or download stuff from this stage?